db/migrate/20150413043822_add_settings_to_instances_and_courses.rb

```diff
@@ -0,0 +1,6 @@
+class AddSettingsToInstancesAndCourses < ActiveRecord::Migration
+  def change
+    add_column :instances, :settings, :text
```

Comment:
OK, further checked, <code>{}</code> is null in database, so this column is not null false.